README.md

```diff
@@ -199,6 +202,43 @@ deepEqual(result, {
     },
   ],
 });
+
+class Point {
+  line: number;
+  column: number;
+  constructor(line: number, column: number) {
```

Comment:
This is TypeScript syntax inside a JavaScript code block. This causes an ESLint parsing error. Could you either strip the type definitions or change the code block type to `ts`?